src/view/com/util/post-embeds/ExternalLinkEmbed.tsx

```diff
@@ -81,10 +70,8 @@ export const ExternalLinkEmbed = ({
               }}
               source={{uri: imageUri}}
               accessibilityIgnoresInvertColors
-              accessibilityLabel={starterPackParsed ? link.title : undefined}
-              accessibilityHint={
-                starterPackParsed ? _(msg`Navigate to starter pack`) : undefined
-              }
+              accessibilityLabel={link.title}
```

Comment:
oh wait, this should now be `undefined` right? i’m confused about the purpose of this prop now